etc/settings.yaml.in

```diff
@@ -1,25 +1,66 @@
 locations:
-  input-dir: in/
-  output-dir: out/
-  genome-dir: genome/
+  input-dir: /path/to/reads/
+  output-dir: /path/to/output/
+  genome-fasta: /path/to/genome.fasta
 
 general:
-  assembly: hg19
+  assembly: ''
+  # use one or both bisulfite aligners
+  # bwameth is faster and uses less resources
+  use_bwameth: False
+  # bismark is gold standard and uses sensible defaults
+  use_bismark: True
   methylation-calling:
     minimum-coverage: 10
     minimum-quality: 10
+    # this applies to bwameth only
+    keep-singleton: False
+    # keep-dups can be Auto/True/False, Auto decides based on protocol
+    keep-Dups: Auto                 
+  export-bigwig:
+    # extract/export contexts other than CpG? 
+    context:
+      cpg: 
+        export: True
+        # join both strands and summarize coverage/methylation? 
+        destrand: True
+      chg: 
+        export: False
+        destrand: False
+      chh: 
+        export: False
+        destrand: False
```

Comment:
I don't have a strong opinion on this, but it may be better to either invert the hierarchy here:

    context: export: [cpg, chg, chh] destrand: []

Or to use the tokens directly instead of booleans:

    context: cpg: [export, destrand] chg: [] chh: []